guppylang/std/qsystem/__init__.py

```diff
@@ -17,6 +17,19 @@
 @guppy
 @no_type_check
 def phased_x(q: qubit, angle1: angle, angle2: angle) -> None:
+    r"""phased_x gate command.
+
+    .. math::
+
+        \mathrm{PhasedX}(\theta_1, \theta_2)=
+          \mathrm{Rz(\theta_2)Rx(\theta_1)Rz(\theta_2)}&=
```

Comment:
```suggestion
          \mathrm{Rz(\theta_2)Rx(\theta_1)Rz(-\theta_2)}&=
```
This was marked as resolved but the change was not made.